bundles/framework/layerselector2/Flyout.js

```diff
@@ -157,7 +167,21 @@ Oskari.clazz.define('Oskari.mapframework.bundle.layerselector2.Flyout',
                 },
                 'featuredata');
         },
+        /**
+         * Add timerseries filter.
+         * @method  @public addNewestFilter
+         */
+        addTimeseriesFilter: function() {
```

Comment:
This should be in the timeseries bundle.

Reply:
A better place for the similar addFeaturedataFilter() should be on the mapwfs bundle. Maybe the modelbuilder class in it or the plugin.